apps/react-storybook/stories/utils/wrappers.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useRef, useEffect } from "react";
+import * as inferno from "inferno";
+
+export function InfernoToReact<TProps>(component) {
+  return function WrappedInfernoComponent(props: TProps) {
+    const ref = useRef();
+    useEffect(() => {
+      inferno.render(
+        inferno.createComponentVNode(0, component, props as any),
+        ref.current
+      );
+    });
+  
+    return <div ref={ref} />;  
+  }
+}
+
+export function DxToReact<TProps>(component) {
```

Comment:
Same thing, consider these renamings:

```suggestion
export function asJqInReact<TProps>(component) {
```
or
```suggestion
export function wrapJqWithReact<TProps>(component) {
```